apps/sports-helsinki/src/domain/search/combinedSearch/CombinedSearch.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  EventTypeId,
+  useLocale,
+  useSearchTranslation,
+} from 'events-helsinki-components';
+import { useRouter } from 'next/router';
+import qs from 'query-string';
+import React from 'react';
+import { ContentContainer, PageSection } from 'react-helsinki-headless-cms';
+import { ROUTES } from '../../../constants';
+import { getI18nPath } from '../../../utils/routerUtils';
+import type { AdvancedSearchProps } from '../eventSearch/AdvancedSearch';
+import AdvancedSearch from '../eventSearch/AdvancedSearch';
+import styles from './combinedSearch.module.scss';
+import { SearchTab } from './searchUtilities/SearchTabs';
+import SearchUtilities from './searchUtilities/SearchUtilities';
+
+export type SearchResultItemsCount = {
+  [EventTypeId.Course]: number | null;
+  [EventTypeId.General]: number | null;
+  Venue: number | null;
+};
+
+export type CombinedSearchProps = {
+  searchStatus: SearchResultItemsCount;
+} & AdvancedSearchProps;
+
+const CombinedSearch: React.FC<CombinedSearchProps> = ({
```

Comment:
I think this component would benefit the compounded components pattern as has kinda compounded/combined nature: 
https://blog.logrocket.com/understanding-react-compound-components/
With the pattern it could look something like:
```
<CombinedSearch>
   <AdvancedSearch ... />
   <Tabs>
      <SearchTab .../>
      <SearchTab .../>
    </Tabs>
    {/* other utilities like a venues map search switcher */}
<CombinedSearch>    
```
It might be easier to maintain and re-use?

Reply:
Sound interesting and this could definitely be tried. I just didn't fully understand the concept yet. I need to read more.

Reply:
I'll continue doing this in a new PR